backend/UnicornRewards.API.Models/Models/User.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace UnicornRewards.API.Models.Models
+{
+    public class User
```

Comment:
Use PascalCase for property names as a standard for C# unless otherwise indicated, this applies for everything in Unosquare